commit e329fd3a0c728443e712133a4297cc5be3fd4fa2
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Mon Oct 8 12:20:36 2018 +0100

    BCM2708_DT: Use upstreamed GPIO expander driver
    
    The upstreamed driver for the GPIO expander has a different compatible
    string. Change the relevant Device Tree files to match.
    
    See: https://github.com/raspberrypi/linux/issues/2704
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts b/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
index 7821483f7d50..fa95f4e3653f 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
@@ -84,7 +84,7 @@
 
 &soc {
 	expgpio: expgpio {
-		compatible = "brcm,bcm2835-expgpio";
+		compatible = "raspberrypi,firmware-gpio";
 		gpio-controller;
 		#gpio-cells = <2>;
 		firmware = <&firmware>;
diff --git a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
index 3f84e2af8c25..951292d2ad16 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
@@ -92,7 +92,7 @@
 	};
 
 	expgpio: expgpio {
-		compatible = "brcm,bcm2835-expgpio";
+		compatible = "raspberrypi,firmware-gpio";
 		gpio-controller;
 		#gpio-cells = <2>;
 		firmware = <&firmware>;
diff --git a/arch/arm/boot/dts/bcm2710-rpi-cm3.dts b/arch/arm/boot/dts/bcm2710-rpi-cm3.dts
index 2500641c14dc..1103b5465521 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-cm3.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-cm3.dts
@@ -52,7 +52,7 @@
 	};
 
 	expgpio: expgpio {
-		compatible = "brcm,bcm2835-expgpio";
+		compatible = "raspberrypi,firmware-gpio";
 		gpio-controller;
 		#gpio-cells = <2>;
 		firmware = <&firmware>;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1612_e329fd3a0c728443e712133a4297cc5be3fd4fa2.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1612_e329fd3a0c728443e712133a4297cc5be3fd4fa2.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
